api/tests/integration/infrastructure/repositories/organization-repository_test.js

```diff
@@ -474,6 +480,18 @@ describe('Integration | Repository | Organization', function () {
       expect(foundOrganization[0].type).to.equal(organizationSCO.type);
       expect(foundOrganization[0].email).to.equal(organizationSCO.email);
     });
+
+    it('should return archivedAt attribute', async function () {
+      // given
+      const uai = '1234568';
+      const organizationSCO = organizations[1];
+
+      // when
+      const foundOrganization = await organizationRepository.findScoOrganizationByUai({ uai });
+
+      // then
+      expect(foundOrganization[0].archivedAt).to.deep.equal(organizationSCO.archivedAt);
+    });
```

Comment:
**suggestion**
On pourrait le mettre dans un contexte `"when organization is archived"` et, dans le cas standard, vérifier qu'on renvoie null quand l'organisation n'est pas archivée.